lib/security_policy.c

```diff
@@ -133,8 +134,7 @@ security_policy_authentication (
 	 * EFI_SECURITY_VIOLATION */
 	fail_status = efi_status;
 
-	efi_status = uefi_call_wrapper(BS->LocateDevicePath, 3,
-				       &SIMPLE_FS_PROTOCOL, &DevPath, &h);
+	efi_status = LocateDevicePath(&SIMPLE_FS_PROTOCOL, &DevPath, &h);
```

Comment:
I don't understand how you are reaching LocateDevicePath here without going through gBS->LocateDevicePath.